cli_tools/gce_vm_image_import/importer/bootable_disk_processor_test.go

```diff
@@ -16,25 +16,41 @@ package importer
 
 import (
 	"bytes"
+	"fmt"
 	"os"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestBootableDiskProcessor_Process(t *testing.T) {
```

Comment:
```suggestion
func TestBootableDiskProcessor_Process_WritesSourceDiskVar(t *testing.T) {
```

Reply:
Renamed to ProcessUEFI since "WritesSourceDiskVar" is one thing to validate. There can be more var to validate